apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/PageController.java

```diff
@@ -61,26 +70,36 @@ public ResponseEntity<Resource> downloadPageLibrary() throws IOException {
     InputStreamResource file = new InputStreamResource(pageDownloader.downloadLibrary());
 
     return ResponseEntity.ok().header(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=" + fileName)
-        .contentType(MediaType.parseMediaType("application/csv")).body(file);
+            .contentType(MediaType.parseMediaType("application/csv")).body(file);
   }
 
   @GetMapping("downloadPDF")
   public ResponseEntity<byte[]> downloadPageLibraryPDF() throws DocumentException, IOException {
     var pdf = pageDownloader.downloadLibraryPDF();
 
     return ResponseEntity.ok()
-        .contentType(MediaType.APPLICATION_PDF)
-        .header(HttpHeaders.CONTENT_DISPOSITION,
-            ContentDisposition.attachment()
-                .filename("PageLibrary.pdf").build()
-                .toString())
-        .body(pdf);
+            .contentType(MediaType.APPLICATION_PDF)
+            .header(HttpHeaders.CONTENT_DISPOSITION,
+                    ContentDisposition.attachment()
+                            .filename("PageLibrary.pdf").build()
+                            .toString())
+            .body(pdf);
 
   }
 
   @DeleteMapping("{id}/delete-draft")
-  public PageStateResponse deletePageDraft(@PathVariable("id") Long pageId) {
-    return stateChange.deletePageDraft(pageId);
+  public PageDeleteResponse deletePageDraft(@PathVariable("id") Long pageId) {
+    return pageDeletor.deletePageDraft(pageId);
+  }
+
+  @GetMapping("{waTemplateUid}/download-metadata")
+  public ResponseEntity<Resource> downloadPageMetadata(@PathVariable("waTemplateUid") Long waTemplateUid) throws IOException {
+    String fileName = "PageMetadata.xls";
```

Comment:
Suggestion: NBS Classic created an `xls` while the newer library is creating a `xlsx` file.  Since the file the API is creating is `xlsx` but has `xls` extension, I am getting a warning when opening it. 

I recommend changing the file name to `xlsx`. This will differ from Classic but makes sense as that is the newer format.